src/modules/roc_core/target_stdio/roc_core/log.h

```diff
@@ -36,11 +36,21 @@ enum LogLevel {
     LogTrace  //!< Debug message (extra verbosity).
 };
 
+//! Colors mode.
+enum ColorsMode {
```

Comment:
Could you please move this inside the `core` namespace? We usually don't add things to the root `roc` namespace, LogLevel is one of the few exceptions.